torch/_dynamo/symbolic_convert.py

```diff
@@ -1258,6 +1259,11 @@ def BUILD_LIST(self, inst):
         options = VariableTracker.propagate(items)
         self.push(ListVariable(items, mutable_local=MutableLocal(), **options))
 
+    def BUILD_SET(self, inst):
+        items = self.popn(inst.argval)
+        options = VariableTracker.propagate(items)
+        self.push(SetVariable(items, mutable_local=MutableLocal(), **options))
```

Comment:
This isn't de-duplicating items in set.

Reply:
You are correct, we should make an empty one and call `add` on each element.